templates/pages/admin/form/form_comment.html.twig

```diff
@@ -0,0 +1,165 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2024 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{# @var form                   Glpi\Form #}
+{# @var comment                Glpi\Form\Comment|null #}
+{# @var section                Glpi\Form\Section|null #}
+{# @var can_update             bool #}
+
+{% import 'components/form/fields_macros.html.twig' as fields %}
+
+{% set base_field_options = {
+    'is_horizontal': false,
+    'full_width'   : true,
+    'no_label'     : true,
+} %}
+
+<div
+    class="mb-3"
+    data-glpi-form-editor-block
+    data-glpi-form-editor-comment
+>
+    <div
+        data-glpi-form-editor-on-click="set-active"
+        data-glpi-form-editor-comment-details
+        class="card"
+    >
+        <div
+            class="card-status-start bg-primary"
+            data-glpi-form-editor-active-comment-status-indicator
+        ></div>
+        <div class="card-body">
+            <div class="d-flex">
+                <i
+                    class="glpi-form-editor-question-handle ti ti-grip-horizontal cursor-grab ms-auto me-auto mt-n3 mb-n2"
+                    data-glpi-form-editor-question-handle
+                    draggable="true"
+                ></i>
+            </div>
+            {# Display comment's title #}
+            <div class="d-flex mt-n1 mb-3 align-items-center">
+                <input
+                    class="form-control content-editable-h2 mb-0"
+                    type="text"
+                    name="name"
+                    value="{{ comment is not null ? comment.fields.name|e('html_attr') : '' }}"
```

Comment:
We decided earlier this week to not longer use this filter as it was unnecessary in the end.

```suggestion
                    value="{{ comment is not null ? comment.fields.name : '' }}"
```